classes/class-wpcom-liveblog-rest-api.php

```diff
@@ -413,7 +413,12 @@ public static function crud_entry( WP_REST_Request $request ) {
 			'headline'        => self::get_json_param( 'headline', $json ),
 		);
 
-		// Add author to contributor list
+		// Ensure contributor_ids is an array.
+		if ( ! is_array( $args['contributor_ids'] ) ) {
```

Comment:
If this isn't an array, what is it? False? WP_Error? Integer? Should we have a more explicit check?

Reply:
@paulschreiber This was `false`.  The reason I added a very broad check here is that directly below we try to push a value into this array, and if it is anything but an array, it will fatal.